modules/raft/pom.xml

```diff
@@ -161,6 +160,11 @@
                             <artifactId>ignite-network-annotation-processor</artifactId>
                             <version>${project.version}</version>
                         </path>
+                        <path>
+                            <groupId>org.openjdk.jmh</groupId>
+                            <artifactId>jmh-generator-annprocess</artifactId>
```

Comment:
Can we use this processor for test sources only?

Reply:
Do you know how to do that?

Reply:
Fixed